src/withPayloadReducer.js

```diff
@@ -1,13 +1,16 @@
+import { identity } from 'lodash';
```

Comment:
To keep the library lightweight, we import individual functions from lodash as a convention. What do you think?

```js
import identity from 'lodash/identity';
```